packages/@remirror/preset-embed/src/iframe-extension.ts

```diff
@@ -36,7 +36,7 @@ export type IframeAttributes = ProsemirrorAttributes<{
   allowFullScreen?: 'true';
   width?: string | number;
   height?: string | number;
-  type?: 'custom' | 'youtube';
+  type?: 'youtube' | string;
```

Comment:
You can use `LiteralUnion` from `type-fest` which allows for autocompletion of the `youtube` string value.

```ts
type?: LiteralUnion<'youtube', string>;
```

Reply:
Good one! Fixed via: https://github.com/remirror/remirror/pull/849/commits/f397e2861781dcc897a6919d3b19d21ac73acd2f